JS-101/andrea-total-a-pagar.html

```diff
@@ -0,0 +1,47 @@
+<script type="text/javascript">
+    /*
+    programa que calcula el total a pagar por un tipo de producto
+    - pregunta precio
+    - pregunta la cantidad de articulos
+    - calcula el subtotal
+    - desglosa el iva
+    - entrega en la consola el total a pagar
+    
+    Entradas: precio, cantidad de productos,
+    Proceso: cantprod x precio, esp x 16%, suma
+    Salida: 
+        GRACIAS POR TU COMPRA
+        subtotal: $000,000.00
+        iva: $000,000.00
+        total: $000,000.00
+    */
+
+    var price;
```

Comment:
All the variables should be aligned with the rest of the file.
Remove extra tab